clang/test/CodeGen/X86/sse-builtins.c

```diff
@@ -34,6 +34,60 @@ __m128 test_mm_andnot_ps(__m128 A, __m128 B) {
   return _mm_andnot_ps(A, B);
 }
 
+__m128 test_mm_cmp_ps_eq_oq(__m128 a, __m128 b) {
+  // CHECK-LABEL: test_mm_cmp_ps_eq_oq
+  // CHECK: fcmp oeq <4 x float> %{{.*}}, %{{.*}}
+  return _mm_cmp_ps(a, b, _CMP_EQ_OQ);
```

Comment:
I don't understand why moving them here. Shouldn't they emit error without avx?

Reply:
These condition codes are [0,7], so they won't throw errors. What about adding 8 more tests to avx-builtins.c with condition codes of [8,15]?

Reply:
Oh the remaining 24 are already there. So I only moved the first 8 from avx to sse/sse2. which is expected.